tests/plugins/clipboard_figure.test.ts

```diff
@@ -153,3 +155,143 @@ describe("Clipboard for figures", () => {
     });
   });
 });
+
+// --------------------------------------------
+
+describe("Clipboard for image figures", () => {
```

Comment:
Instead of copy/pasting the whole describe and have one describe for charts, and one for images that have 90% the same content, you can probably merge both.

Something like `describe.each(["chart", "image"]){...}`, and then helpers that take as argument the figure type ("chart" | "image"), to create/get the figure.